src/components/button-swap/button-swap.twig

```diff
@@ -0,0 +1,28 @@
+{#
+  This pattern currently only supports a start icon.
+  End icon support can be added in the future if needed.
+#}
+<section role="region" data-testid="test-id" class="c-button-swap js-c-button-swap{% if class %} {{ class }}{% endif %}">
+  <div class="c-button-swap__subscribe js-c-button-swap__first-btn-wrapper">
+    <span tabindex="-1" role="status" class="u-hidden-visually js-c-button-swap__message">
+      {{ first_button_visually_hidden_text|default('Unsubscribed from notifications') }}
```

Comment:
Nit: I worry a bit that someone might mishear this as "Unsubscribe" instead of "Unsubscribed"

I wonder if adding "Currently" to the stat of these messages might make them clearer?

Reply:
Done! https://github.com/cloudfour/cloudfour.com-patterns/pull/1516/commits/32da47af9c44fcc0927c62fd0ea699bb379c9b72